src/components/safe-apps/SafeAppsSection/CollapsibleSection.tsx

```diff
@@ -0,0 +1,81 @@
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import Typography from '@mui/material/Typography'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import Grid from '@mui/material/Grid'
+import { AppCard } from '@/components/safe-apps/AppCard'
+import { SectionProps } from './types'
+import { AddCustomAppCard } from '@/components/safe-apps/AddCustomAppCard'
+
+const CollapsibleSection = ({
+  title,
+  apps,
+  onPinApp,
+  prependAddCustomAppCard,
+  onAddCustomApp,
+  pinnedIds,
+  cardVariant,
+}: SectionProps) => {
+  const columnSpacing = cardVariant === 'compact' ? 3 : 2
+
+  return (
+    <Accordion
+      sx={{
+        boxShadow: 'none',
+        border: 'none',
+        background: 'transparent',
+      }}
+      defaultExpanded
+    >
+      <AccordionSummary
+        expandIcon={<ExpandMoreIcon />}
+        sx={({ spacing }) => ({
+          padding: `0 ${spacing(3)}`,
+          justifyContent: 'flex-start',
+          background: 'transparent',
+          '& > .MuiAccordionSummary-content': {
+            flexGrow: 0,
+            '&.Mui-expanded': {
+              margin: `${spacing(2)} 0`,
+            },
+          },
+          '&.Mui-expanded': {
+            minHeight: 48,
+          },
+        })}
+      >
+        <Typography variant="caption" fontWeight={700} sx={({ palette }) => ({ color: palette.secondary.light })}>
+          {title}
+        </Typography>
+      </AccordionSummary>
+      <AccordionDetails sx={({ spacing }) => ({ padding: `0 ${spacing(3)}` })}>
```

Comment:
Side note: we have CSS vars for spacing if you ever want to do this in CSS modules themselves.

Reply:
yep, have seen it. I personally prefer this one-liner instead of creating a separate file and importing it, so I'd keep this line as is if you don't mind

Reply:
Sure, was just letting you know